franklab_scripts/alter_tables.py

```diff
@@ -0,0 +1,39 @@
+"""Update the MySQL tables with the latest table definitions in each nwb_datajoint.common class."""
```

Comment:
have you tested that calling this doesn't affect the data already in the table?

Reply:
Yes, but I will double check using a more recent DB dump.

Reply:
For the record, all schema changes fall into one of the following categories:
1) add or modify a default value for a column
2) allow a foreign key to be nullable
3) add an enum value, e.g. "Unknown"
4) expand varchar length
5) reduce varchar length for object IDs to 40 -- if the affected object_id is > 40 characters, then the alter command will raise an error, but the object ID should not be > 40 characters.

Reply:
I checked all of the ALTER SQL commands issued by DataJoint on a DB dump from yesterday. Primary keys cannot be altered (DataJoint silently ignores attempts to do so rather than returning an error), so I have updated the code here to not alter them. 

There are two tables in the DB that I did not change but running `alter` on them revealed that they were out of sync with the codebase. I will post about this in the main PR discussion.